runtime/src/main/java/org/corfudb/runtime/view/Address.java

```diff
@@ -38,6 +38,12 @@ public static boolean isAddress(long addr) {
     @Getter
     private static long minAddress = 0L;
 
+    /**
+     * Return the first address higher than the minAddress so sequence is never 0.
```

Comment:
Why sequence should be greater than 0?

Reply:
It's really helpful in sanity checking the sequence and also for testing. I added a comment explaining this.